test/runtests.jl

```diff
@@ -21,7 +21,7 @@ for (s, enc) in (("noël", "ISO-8859-1"),
 end
 
 # Test that attempt to close stream in the middle of incomplete sequence throws
-let s = "a string チャネルパートナーの選択", a = Vector{UInt8}(s)
+let s = "a string チャネルパートナーの選択", a = unsafe_wrap(Vector{UInt8}, s)
```

Comment:
I think that is the right fix, but now tests fail...

Reply:
I don't think anything needs to be changed, IIRC this will now make a copy but it's OK.